src/libraries/icubmod/cfw2Can/Cfw2Can.cpp

```diff
@@ -171,7 +171,7 @@ bool Cfw2Can::canGetErrors(CanErrors &err)
 
 CanMessage &Cfw2CanMessage::operator=(const CanMessage &l)
 {
-    const Cfw2CanMessage &tmp=dynamic_cast<const Cfw2CanMessage &>(l);
```

Comment:
I would add a comment to remind us that reinterpret_cast is used for efficiency but may hide bugs if the wrong object is passed on the call to Cfw2CanMessage::operator=(const CanMessage &l)